src/components/subscriptions/LicenseAllocationNavigation.jsx

```diff
@@ -36,22 +48,37 @@ export default function LicenseAllocationNavigation() {
   );
 
   return (
-    <nav className="nav sticky-top">
-      <ul className="list-unstyled w-100">
-        {tabs.map(tab => (
-          <li key={tab.key}>
-            <button
-              className={classNames(
-                'btn btn-link btn-block pl-0 text-left',
-                { 'font-weight-bold': activeTab === tab.key },
-              )}
-              onClick={() => setActiveTab(tab.key)}
-            >
-              {tab.text}
-            </button>
-          </li>
-        ))}
-      </ul>
-    </nav>
+    <React.Fragment>
+      {isLoading && <LoadingMessage className="loading mt-3" />}
+      {
+        error && <StatusAlert
```

Comment:
As is, if both the overview API call and the list API call error, through this PR, we'd be showing 2 error status alerts? One in the sidebar and one in the table area? 

I'm wondering if it makes sense to show a single error status alert if either of those calls fail rather than showing potentially 2 alerts right next to each other, especially with one being in a small sidebar that would result in a narrow width for the status alert.

Reply:
I have moved error messages to a single place, i.e. above the grid.